src/main/java/seedu/address/storage/JsonAddressBookStorage.java

```diff
@@ -31,6 +32,30 @@ public Path getAddressBookFilePath() {
         return filePath;
     }
 
+    /**
+     * Similar to {@link #readAddressBook()}.
+     *
+     * @param filePath location of the data. Cannot be null.
+     * @throws DataConversionException if the file is not in the correct format.
+     */
+    public Optional<ReadOnlyHotel> readHotel(Path filePath) throws DataConversionException {
+        requireNonNull(filePath);
+
+        Optional<JsonSerializableHotel> jsonHotel = JsonUtil.readJsonFile(
+                filePath, JsonSerializableHotel.class);
+        if (!jsonHotel.isPresent()) {
+            System.out.println("hotel empty");
```

Comment:
Could you use logger for this? Use warning that the hotel is not found (not "hotel empty" right?) .

Reply:
This was the original code, hence I didn't figure out how to use logger

Reply:
```suggestion
           logger.warning("Hotel not found");
```